build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -62,10 +62,10 @@ class DriverInfo:
 # mapping to more friendly ones and detach to keep driver names used in
 # benchmark presentation stable.
 IREE_DRIVERS_INFOS = {
-    "iree-dylib":
-        DriverInfo("IREE-Dylib", "CPU", "local-task", "embedded-elf"),
-    "iree-dylib-sync":
-        DriverInfo("IREE-Dylib-Sync", "CPU", "local-sync", "embedded-elf"),
+    "iree-cpu":
+        DriverInfo("IREE-CPU", "CPU", "local-task", "embedded-elf"),
+    "iree-cpu-sync":
+        DriverInfo("IREE-CPU-Sync", "CPU", "local-sync", "embedded-elf"),
```

Comment:
We're discussing alternate "friendly" and "config" names for these over on Discord here: https://discord.com/channels/689900678990135345/689906000043573354/1001988964921655306

Reply:
Yeah we'd need to update the website's database to match the new friendly names to avoid creating new series. @pzread I assume you know how this should be done now? Do you want to do that? I can do it otherwise.

Reply:
Run a script like the following in the `dana/configs/db/` directory should largely do the work I believe:

```py
#!/usr/bin/env python

import os
import re

path = os.getcwd()
files = os.listdir(path)

for old_name in files:
    new_name = re.sub(
        r'IREE-Dylib',  r'IREE-CPU', old_name)
    os.rename(os.path.join(path, old_name), os.path.join(path, new_name))
```

We'd need to do that immediately before landing this pull request.

Reply:
Renamed to `IREE-LLVM-CPU`/`IREE-LLVM-CPU-Sync` per @benvanik 's suggestions on Discord (disambiguating between VMVX executables, which also run on the CPU, and the llvm-cpu codegen executables)

Reply:
I can do this, but we need to coordinate: 1. stop the dashboard server 2. merge this PR 3. backfill the database 4. start the dashboard server.

This backfill will also make others PR stopped showing the correct benchmark summary if they don't rebase. It's not a big issue, but it's better to let other people know about this.

Reply:
Awesome, thanks! Please also make sure that 1) the name looks good once `buildkite:benchmark` posts here, 2) zip and backup the existing `db/` before updating names, 3) double check names in the website looks good and the same as the comments on the pull request after updating. It's typically easier to do it early in the morning to make sure no pending builds and land this pull request immediately after updating the database.